frontend/src/context/ApiCallsContext.jsx

```diff
@@ -39,6 +37,7 @@ export function ApiCallsContextProvider({ children }) {
           .sort(() => Math.random() - 0.5),
       });
     });
+    movieIdArray.push(movieIdArray.splice(0, 1));
```

Comment:
With this we lose the notion of randomness during the game. You have to move four cards and not one! Try with a state and by making a shift of 4 "notches" in your table for each question.